sympy/functions/elementary/complexes.py

```diff
@@ -662,7 +662,7 @@ def _eval_nseries(self, x, n, logx, cdir=0):
         if direction.has(log(x)):
             direction = direction.subs(log(x), logx)
         s = self.args[0]._eval_nseries(x, n=n, logx=logx)
-        return (sign(direction)*s).expand()
+        return (s/sign(direction)).expand()
```

Comment:
It seems that this will be infinite

Reply:
I was concerned about this, too. I suspect 0 doesn't get here, but if it did then
```python
s = ...
if direction.is_extended_positive or direction.is_extended_negative:
    s = s/sign(direction)
elif direction.is_zero:
    s = S.Zero
else:
    raise ValueError('sign of direction unknown')
```

Reply:
Yeah as pointed out above atleast for the current set of tests, we might not encounter a `0` but yeah that should be taken care of. Rest the change looks justified to me. @saicharan2804 could you take care of this corner case.